tunnel/config.go

```diff
@@ -50,13 +51,20 @@ func (r SSHConfigFile) Get(host string) *SSHHost {
 		user = ""
 	}
 
+	localForward, err := r.getLocalForward(host)
+	if err != nil {
+		localForward = &LocalForward{Local: "", Remote: ""}
+		log.Debugf("error to read LocalForward configuration from ssh config file")
```

Comment:
I think the proper log level here should be warning instead of debug, since it is  some sort of error, but won't make the application to stop working.
The message should include the error received and explicit say the error occurred, but it will try to continue to work without using the LocalForward option from the ssh config file.

```suggestion
		log.Warningf("error reading LocalForward configuration from ssh config file. This option will not be used: $v", err)
```